sass/components/_CreateVmWizard.scss

```diff
@@ -1,3 +1,7 @@
 .modal .wizard-pf-main {
   padding-top: 10px;
 }
+
+#cloud-init-custom-script {
```

Comment:
I am not strictly against it, but prefer use of classes instead of ids

Reply:
if I understand it correctly, BEM also prefers classes instead of IDs. In any case, please also add kubevirt prefix

Reply:
+1, I will update this PR to the recent standard